index.js

```diff
@@ -20,6 +21,7 @@ if (!isNotBrowser) {
   isSafari = Object.prototype.toString.call(window.HTMLElement).indexOf('Constructor') > 0 || (window.safari && window.safari.pushNotification);
   isOpera = (!!window.opr && !!window.opr.addons) || !!window.opera;
   isEdge = !!window.StyleMedia;
+  isEdgeChromium = isChromeOrChromium && !!navigator.userAgent.match(/edg/i);
```

Comment:
Used the same SO for this one: https://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser